src/book/02-system/01-foundations/02-build-system.mdx

```diff
@@ -344,6 +345,44 @@ The `nuc7i7bnh` target should be used for building and running code on the robot
 
 The `target` command will download (or, if needed, build) the Docker image for the specified target and then mark that target as active.
 
+##### Tests
+
+The `tests` command allows you to run built unit tests. It will automatically run tests in parallel, and dump a timestamped log file to the project directory.
+Currently, we use `CTest` which is included as part of `CMake`.
```

Comment:
```suggestion
The `tests` command allows you to run built unit tests. It will automatically run tests in parallel, and dump a timestamped log file to the project directory. Currently, we use [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) which is included as part of CMake.
```